### DISCUSSION TITLE
deleteAllTriggers is not defined

### DISCUSSION THREAD

**Discussion by yelserp87** [derekantrican/GAS-ICS-Sync#432] [Q&A] [answered]
I am getting an error when I run the code:

Error	ReferenceError: deleteAllTriggers is not defined
install	@ Code.gs:102

Any ideas?

**Comment by jonas0b1011001** [comment, marked as answer]
Looks like you are missing a file. Did you manually install or make a copy of the script from [here](https://script.google.com/d/1BOk8MDLbLaHh6SwG1M1tsgNXjkcC-79LE0QoipRuTDxbO3fMVvqoROQD/edit?newcopy=true)?

**Comment by yelserp87** [comment]
Yup that was it. I just copied the code and missed the other files! It's working now thanks!
